packages/scripts.nix

```diff
@@ -159,12 +159,13 @@
     ];
     text = ''
       imageRef=$1
+      platform=$2
 
       tmpdir=$(mktemp -d)
       trap 'rm -rf $tmpdir' EXIT
 
       echo "ghcr.io/edgelesssys/contrast/coordinator:latest=$imageRef" > "$tmpdir/image-replacements.txt"
-      resourcegen --image-replacements "$tmpdir/image-replacements.txt" --add-load-balancers coordinator > "$tmpdir/coordinator_base.yml"
+      resourcegen --image-replacements "$tmpdir/image-replacements.txt" --add-load-balancers --platform $platform coordinator > "$tmpdir/coordinator_base.yml"
```

Comment:
The coordinator is still platform independent, right? I think this snippet shouldn't take platform input.